src/bootstrap-tenant/__dev__bootstrap.ts

```diff
@@ -126,7 +126,7 @@ async function bootstrap() {
     })
 
     // bootstrapper.config.itemTopics = 'amend'
-    // bootstrapper.config.logLevel = 'verbose'
+    bootstrapper.config.logLevel = 'verbose'
```

Comment:
Is this supposed to be commented out?

Reply:
It's just for development purposes. Maybe it makes sense to have it commented out by default.